src/components/InngestCommHandler.ts

```diff
@@ -427,8 +427,8 @@ export class InngestCommHandler<H extends Handler, TransformedRes> {
     }
 
     const prefix =
-      this.signingKey.match(/^signkey-(test|prod)-/)?.shift() || "";
-    const key = this.signingKey.replace(/^signkey-(test|prod)-/, "");
+      this.signingKey.match(/^signkey-(test|prod|branch)-/)?.shift() || "";
```

Comment:
Should this be more general?

```suggestion
      this.signingKey.match(/^signkey-[a-z]+-/)?.shift() || "";
```

Reply:
Yeah, it's a nit, but `[a-z]+` or `\w+` feels reasonable.